setup.cfg

```diff
@@ -1,3 +1,60 @@
+[metadata]
+name = pytype
+description = Python type inferencer
+version: attr:pytype.__version__.__version__
+long_description_content_type = text/markdown
+maintainer = Google
+maintainer_email = pytype@googlegroups.com
+url = https://google.github.io/pytype
+project_urls =
+    Code = https://github.com/google/pytype
+    Issue Tracker = https://github.com/google/pytype/issues
+    Documentation = https://google.github.io/pytype
+license = Apache 2.0
+license_file = LICENSE
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    License :: OSI Approved :: Apache Software License
+    Programming Language :: Python
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Software Development
+
+
+[options]
+zip_safe = False
+python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.8
+packages =
+    find:
+install_requires =
+    attrs
+    importlab>=0.5.1
+    ninja
+    pyyaml>=3.11
+    six
+    typed_ast; python_version >= "3.3"
+
+[options.packages.find]
+include =
+    pytype
+    pytype.*
+exclude =
+    pytype.test_data
+    pytype.tests
+    pytype.typeshed.tests
+
+[options.entry_points]
+console_scripts =
+    annotate-ast = pytype.tools.annotate_ast.main:main
+    merge-pyi = pytype.tools.merge_pyi.main:main
+    pytd = pytype.pytd.main:main
+    pytype = pytype.tools.analyze_project.main:main
+    pytype-single = pytype.single:main
+    pyxref = pytype.tools.xref.main:main
+
 [pytype]
 python_version = 2.7
```

Comment:
should we change this while we're at it?

Reply:
I don't really understand this part of it, so if you don't mind I'd rather leave that to someone else.